modules/apps/asset/asset-categories-selector-web/src/main/resources/META-INF/resources/js/SelectCategory.es.js

```diff
@@ -63,42 +63,42 @@ function SelectCategory({
 	const selectedNodesRef = useRef(null);
 
 	const handleAddCategoryClick = useCallback(() => {
-		const dialog = Liferay.Util.getWindow(itemSelectorSaveEvent);
-		const footer = dialog.getToolbar('footer');
+		const addButton = document.getElementById('#addButton');
 
-		footer.get('boundingBox').one('#addButton').hide();
+		if (addButton) {
+			addButton.style.display = 'none';
```

Comment:
It has to be `display: none;` because they are explicity hiding it when adding a new vocabulary via the dialog window.
![image](https://user-images.githubusercontent.com/860951/114624308-582e6e80-9c65-11eb-8135-352ffbc26aa6.png)

Reply:
Oh I see. Makes sense :+1: